perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -110,6 +103,20 @@ static void PrintFlags(const bool* FlagArray)
     std::cout << OutputBuffer << '\n';
 }
 
+static size_t ComputeEffectiveAddress(const instruction_operand& Operand, const s16* Registers)
+{
+    size_t Result = 0;
+
+    for (auto Term : Operand.Address.Terms)
```

Comment:
I recommend explicitly taking a reference. `auto& Term`

Reply:
Not used anymore, but noted! 
1cf78e683ce906ab23d5f276eba2560eddd97632